netci.groovy

```diff
@@ -209,6 +216,13 @@ def static addTriggers(def job, def isPR, def architecture, def os, def configur
             case 'forcerelocs':
             case 'jitstress1':
             case 'jitstress2':   
+            case 'js2_jsr1':
```

Comment:
Is there a reason (like path length) that you abbreviated?  If not, I think it should be spelled out.